Userland/Libraries/LibWeb/Infra/Strings.cpp

```diff
@@ -106,4 +106,17 @@ ErrorOr<String> convert_to_scalar_value_string(StringView string)
     return scalar_value_builder.to_string();
 }
 
+ErrorOr<String> to_ascii_lower_case(StringView string)
```

Comment:
`// https://infra.spec.whatwg.org/#ascii-lowercase`

Reply:
Feel free to add it, sure! We'll find a use case soon enough.

Reply:
OK, one sec :)

Reply:
Done